tasks/autorestic.yml

```diff
@@ -1,34 +1,61 @@
 ---
 # autorestic tasks file for autorestic
+- name: Ensure autorestic directory exist
+  ansible.builtin.file:
+    state: directory
+    path: /opt/autorestic/bin
+    mode: 0755
+    owner: root
+
 - name: "Download autorestic binary v{{ autorestic_version }} for aarch64"
```

Comment:
Removing the file permissions causes the Ansible-Lint warning shown here. I think we should just keep the `mode` as it was.

Reply:
0644 should be enough for the downloaded compressed files, as they do not need to be executable. i readd that.